cmake/configs/default.cmake

```diff
@@ -85,7 +85,11 @@ hunter_config(NASM VERSION 2.12.02)
 hunter_config(OpenBLAS VERSION 0.2.20-p0)
 hunter_config(OpenCL VERSION 2.1-p3)
 hunter_config(OpenCL-cpp VERSION 2.0.10-p0)
-hunter_config(OpenCV VERSION 3.2.0-p2)
+if(IOS)
+  hunter_config(OpenCV VERSION 3.2.0-p2)
+else()
+  hunter_config(OpenCV VERSION 3.3.0-p0)
```

Comment:
Is OpenCV-Extra 3.2 compatible with OpenCV 3.3? I see 3.3.0 here:
* https://github.com/ruslo/hunter/blob/master/cmake/projects/OpenCV-Extra/hunter.cmake

Probably they should be updated simultaneously (?)